dashboard/app/models/applab.rb

```diff
@@ -205,7 +210,7 @@ def self.palette
         "functionParams_n": null,
         "callMyFunction": null,
         "callMyFunction_n": null,
-        "return": null
+        "return": null,
```

Comment:
Looks like trailing commas are valid ECMAScript5, but not valid JSON: http://www.json.org/